<launch>
    <!-- Silly shapes shows test visualization_markers -->
    <arg name="silly_shapes" default="false"/>
    
    <include file="$(find vrviz)/launch/vrviz.launch">
      <arg name="cloud_remap" value="/camera/depth_registered/points"/>
      <arg name="marker_remap" value="/marker_array"/>
      <arg name="load_robot" value="true"/>
      <arg name="show_tf" value="true"/>
      <arg name="scaling_factor" value="1"/>
      <arg name="point_size" value="1"/>
      <!-- This does not work due to the camera_info messages not being synced.-->
      <arg name="image_remap" value="/camera/rgb/image_raw"/>
      <!-- <arg name="hud_dist" default="0.5"/>-->
      <arg name="twist_remap" value="/cmd_vel_mux/input/teleop"/>
      <arg name="manual_image_copy" value="true"/>  
  </include>

    <node pkg="tf" type="static_transform_publisher" name="coordinate_vrviz_base_base"
        args="0 0 0 -1.57 0 1.57 camera_link vrviz_base  10" />
        
   <!-- Connect to robot frame -->
   <!-- <node pkg="tf" type="static_transform_publisher" name="coordinate_vrviz_robot"
        args="0 0 0 0 0 1.57079632679 world vrviz_base  10" />
       --> 
    <node pkg="vrviz" type="marker_test" name="marker_test" >
        <param name="silly_shapes" value="$(arg silly_shapes)" />
    </node>
</launch>
